lib/brainstem/presenter_collection.rb

```diff
@@ -250,7 +253,12 @@ def extract_filters(options)
 
         filter_options = filter[0]
         args = run_defaults && requested.nil? ? filter_options[:default] : requested
-        filters_hash[filter_name] = args unless args.nil?
+        include_params = filter_options[:include_params]
+        if param_opts
+          filters_hash[filter_name] = [args, include_params]
+        else
+          filters_hash[filter_name] = args unless args.nil?
```

Comment:
Maybe make this always point to a hash now like `{ args: args, include_params: include_params }`?